index.js

```diff
@@ -5,7 +5,7 @@ module.exports = {
   name: 'ember-stripe-service',
   contentFor: function(type) {
     // we use body since Stripe must be available before
-    if (type === 'body') {
+    if (type === 'body' && process.env.EMBER_CLI_FASTBOOT !== 'true') {
```

Comment:
been a while, but I believe the stripe javascript API assumes the existence of globals we don't have in Fastboot